python/cusignal/reader/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from cusignal.reader.reader import read_bin, parse_bin
```

Comment:
nit: can we change the filename under cusignal to io? Rather than 

`from cusignal.reader.reader`, we can have `from cusignal.io.reader`